### PR TITLE
 Tick format doc for R

### PR DESCRIPTION
**Pull request by priyatharsan**
related to https://github.com/plotly/documentation/issues/1137

### CODE REVIEW

**Comment by priyatharsan** [comment]
@michaelbabyn Looks like there is a bug with `Tickformatstops` in R API, logged an issue here: https://github.com/ropensci/plotly/issues/1486 
Other than that the doc is good I guess.

**Comment by priyatharsan** [comment]
@michaelbabyn  thanks for finding this out. Specifying `type='date'` to the axis fixed the issue

File: _posts/r/layout/tick-format/2019-03-08-tick-formatting.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+library(plotly)
+
+p <- plot_ly(
+  type = "scatter",
+  x = c(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12), 
+  y = c(28.8, 28.5, 37, 56.8, 69.7, 79.7, 78.5, 77.8, 74.1, 62.6, 45.3, 39.9), 
+  mode = "markers+lines") %>%
+  layout(
+    xaxis = list(
+      ticktext = c("One", "Three", "Five", "Seven", "Nine", "Eleven"),
```
Comment (michaelbabyn) [inline _posts/r/layout/tick-format/2019-03-08-tick-formatting.md]: For some reason this isn't showing up on chart studio using the character vector, it works with list though
```suggestion
      ticktext = list("One", "Three", "Five", "Seven", "Nine", "Eleven"), 
```

File: _posts/r/layout/tick-format/2019-03-08-tick-formatting.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+p <- plot_ly(
+  type = "scatter",
+  x = c(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12), 
+  y = c(28.8, 28.5, 37, 56.8, 69.7, 79.7, 78.5, 77.8, 74.1, 62.6, 45.3, 39.9), 
+  mode = "markers+lines") %>%
+  layout(
+    xaxis = list(
+      ticktext = c("One", "Three", "Five", "Seven", "Nine", "Eleven"), 
+      tickvals = c(1, 3, 5, 7, 9, 11),
```
Comment (michaelbabyn) [inline _posts/r/layout/tick-format/2019-03-08-tick-formatting.md]:
```suggestion
      tickvals = list(1, 3, 5, 7, 9, 11),
```

File: _posts/r/layout/tick-format/2019-03-08-tick-formatting.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+      tickmode = "array"
+  ))
+
+# Create a shareable link to your chart
+# Set up API credentials: https://plot.ly/r/getting-started
+chart_link = api_create(p, filename="tickformatting-tickmode-array")
+chart_link
+```
+
+<iframe src="https://plot.ly/~RPlotBot/5616.embed" width="800" height="600" id="igraph" scrolling="no" seamless="seamless" frameBorder="0"> </iframe>
```
Comment (michaelbabyn) [inline _posts/r/layout/tick-format/2019-03-08-tick-formatting.md]: https://plot.ly/~RPlotBot/5616.embed isn't working with the c vectors (see above)

Comment (priyatharsan) [reply _posts/r/layout/tick-format/2019-03-08-tick-formatting.md]: this should work now

Comment (michaelbabyn) [reply _posts/r/layout/tick-format/2019-03-08-tick-formatting.md]: The update looks good. :dancer:

**Pull request merged**